cmake_modules/commonSetup.cmake

```diff
@@ -642,6 +649,7 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
       endif(USE_OPENSSL)
 
       if(USE_MYSQL_REPOSITORY)
+        set(MYSQL_INCLUDE_DIR "")
```

Comment:
This needs to be in the else branch - it will already be set up in this branch, after:

add_definitions (-D_NO_MYSQL_REPOSITORY)